cli/oplog.go

```diff
@@ -9,6 +10,28 @@ import (
 	"github.com/percona/percona-backup-mongodb/pbm"
 )
 
+type replayOptions struct {
+	start string
+	end   string
+	wait  bool
+}
+
+type oplogReplayResult struct {
+	Name string `json:"name"`
+	done bool
+	err  string
+}
+
+func (r oplogReplayResult) String() string {
+	if r.done {
+		return "\nOplog replay successfully finished!\n"
+	}
+	if r.err != "" {
+		return "\n Error: " + r.err
+	}
+	return fmt.Sprintf("Oplog replay %q has started", r.Name)
```

Comment:
`fmt.Stringer` is used to get a string representation of a value that can be used anywhere.
here I see you are using it to format output specifically (~de~coupled) to CLI. I'd use a separate interface/method for CLI output. fallback to `String()` if there is no implementation for CLI output